From 96e8fd94227d422906eca28e8b9733a6411b67f5 Mon Sep 17 00:00:00 2001
From: Paul Grandperrin <paul.grandperrin@gmail.com>
Date: Tue, 17 May 2022 20:20:45 +0200
Subject: [PATCH] fix libultraship compilation

---
 libultraship/libultraship/SDLController.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libultraship/libultraship/SDLController.cpp b/libultraship/libultraship/SDLController.cpp
index ab2c48b..a233903 100644
--- a/libultraship/libultraship/SDLController.cpp
+++ b/libultraship/libultraship/SDLController.cpp
@@ -155,7 +155,7 @@ namespace Ship {
         if (ax != 0.0 && ay != 0.0) {
             auto slope = ay / ax;
             auto edgex = copysign(85.0 / (abs(slope) + 16.0 / 69.0), ax);
-            auto edgey = copysign(std::min(abs(edgex * slope), 85.0 / (1.0 / abs(slope) + 16.0 / 69.0)), ay);
+            auto edgey = copysign(std::min<double>(abs(edgex * slope), 85.0 / (1.0 / abs(slope) + 16.0 / 69.0)), ay);
             edgex = edgey / slope;
 
             auto scale = sqrt(edgex * edgex + edgey * edgey) / 85.0;
-- 
2.36.0

